lib/devise/multi_email/parent_model_extensions.rb

```diff
@@ -9,8 +9,12 @@ module ParentModelExtensions
 
       included do
         multi_email_association.include_module(EmailModelExtensions)
+
+        # Enable "autosave" to ensure email record changes are persisted
+        multi_email_association.reflection.options.merge!(autosave: true)
```

Comment:
@allenwq will keep an eye out for whether this works as expected now that I've removed the setting.